bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -84,6 +88,28 @@ def _cast_experiment_type(self, experiment_type_string: str) -> None:
             raise ValueError('Register this experiment type or remove button')
 
         return experiment_type
+        
+    def load_items_from_txt(self, event):
+        parameters = load_json_parameters('bcipy/parameters/parameters.json', value_cast=True)
```

Comment:
Let's make this location a static variable on line 11 called `PARAMETER_LOCATION = 'bcipy/parameters/parameters.json'`, then here call self.PARAMETER_LOCATION. That will help us if other parameters are wanted.